# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/test/gtest/TestReadWrite.cpp
# Commit: 3399e7c51942
# Full Hash: 3399e7c519424a82824f48abd3b2a4d75346d723
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2019-05-22 03:40:58
# Regressor Bug: 965637
# File Overlap Count: 3
# Description:
#   Bug 965637: Move CSP from Principal into Client, part 2: worker changes. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27655
# ==============================================================================

diff -r ddf4012a7652 -r 3399e7c51942 dom/serviceworkers/test/gtest/TestReadWrite.cpp
--- a/dom/serviceworkers/test/gtest/TestReadWrite.cpp	Tue May 21 23:14:27 2019 +0000
+++ b/dom/serviceworkers/test/gtest/TestReadWrite.cpp	Tue May 21 23:14:50 2019 +0000
@@ -283,10 +283,9 @@
       nsAutoCString spec;
       spec.AppendPrintf("spec write %d", i);
 
-      nsTArray<mozilla::ipc::ContentSecurityPolicy> policies;
       reg.principal() = mozilla::ipc::ContentPrincipalInfo(
           mozilla::OriginAttributes(i % 2), spec, spec, mozilla::Nothing(),
-          std::move(policies), spec);
+          spec);
 
       swr->TestRegisterServiceWorker(reg);
     }
@@ -888,10 +887,9 @@
       nsAutoCString spec;
       spec.AppendPrintf("spec write dedupe/%d", i);
 
-      nsTArray<mozilla::ipc::ContentSecurityPolicy> policies;
       reg.principal() = mozilla::ipc::ContentPrincipalInfo(
           mozilla::OriginAttributes(false), spec, spec, mozilla::Nothing(),
-          std::move(policies), spec);
+          spec);
 
       swr->TestRegisterServiceWorker(reg);
     }